src/router/router.tsx

```diff
@@ -0,0 +1,94 @@
+import type { RouteObject } from 'react-router-dom';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+
+import App from '../App';
+import Authorization from '../components/Authorization';
+import Admin from '../pages/Admin/index';
+import ErrorPage from '../pages/Error/index';
+import HomePage from '../pages/Home/index';
+import LoginPage from '../pages/Login/index';
+import MainPage from '../pages/Main/index';
+import ProductPage from '../pages/Product/index';
+import ReservationPage from '../pages/Reservation/index';
+import UserPage from '../pages/User/index';
```

Comment:
```suggestion
import Admin from '../pages/Admin';
import ErrorPage from '../pages/Error';
import HomePage from '../pages/Home';
import LoginPage from '../pages/Login';
import MainPage from '../pages/Main';
import ProductPage from '../pages/Product';
import ReservationPage from '../pages/Reservation';
import UserPage from '../pages/User';
```

index도 동일하게 생략하는게 일반적입니다
이것도 Node.js와 대부분의 번들러는 디렉토리를 import할 때 자동으로 index 파일을 찾아요~